### PR TITLE
docs(@core/home/quote): add documentation

### PR DESCRIPTION
**Pull request by marobakr**
1. add documentation
if applied, it will make code easier to maintain and readable

### CODE REVIEW

File: src/app/core/components/home/quote/quote.component.ts [hunk 1]
```diff
@@ -7,32 +7,55 @@ import {
...
   imports: [CommonModule],
   templateUrl: './quote.component.html',
-  styleUrl: './quote.component.scss',
+  styleUrls: ['./quote.component.scss'],
 })
 export class QuoteComponent implements OnInit {
-  constructor(private _breakpointObserver: BreakpointObserver) {}
-
+  /**
+   * Flag indicating whether the viewport is in handset portrait mode.
```
Comment (vcpc) [inline src/app/core/components/home/quote/quote.component.ts:23]: put the handset portrait in a codespan to be clear that it is a part of the code

**Comment by vcpc** [comment]
docs not clear and hard to understand at lines: 38, 45, 55, 68, 76 ()

**Comment by marobakr** [comment]
1. add documentation
1.1 wrapping `isHandsetPortrait` flag with (codespan)
1.2 rewrite valid documentation

if applied, it will make code easier to maintain and readable

File: src/app/core/components/home/quote/quote.component.ts [hunk 2]
```diff
@@ -7,32 +7,55 @@ import {
...
   imports: [CommonModule],
   templateUrl: './quote.component.html',
-  styleUrl: './quote.component.scss',
+  styleUrls: ['./quote.component.scss'],
 })
 export class QuoteComponent implements OnInit {
-  constructor(private _breakpointObserver: BreakpointObserver) {}
-
+  /**
+   * Flag indicating whether the viewport in`isHandsetPortrait` mode.
```
Comment (vcpc) [inline src/app/core/components/home/quote/quote.component.ts:23]: The name of the breakpoint is invalid

File: src/app/core/components/home/quote/quote.component.ts [hunk 3]
```diff
@@ -7,32 +7,55 @@ import {
...
+   */
   isHandsetPortrait: boolean = false;
+
+  /**
+   * Flag indicating whether the viewport is in small mode.
```
Comment (vcpc) [inline src/app/core/components/home/quote/quote.component.ts:28]: The name of the breakpoint is invalid

**Comment by marobakr** [comment]
1.add documentation
1.1 wrapping `HandsetPortrait` flag with (codespan)
1.2 wrapping `Small` flag with (codespan)

if applied, it will make code easier to maintain and readable

**Pull request merged**